src/components/reviews/featuresBreakdown.jsx

```diff
@@ -1,10 +1,34 @@
 import React from "react";
+import { MEANINGS } from "../../characteristicMeanings.js";
+import "./reviews.css";
```

Comment:
I think it was a good choice on our team's part to use separate css files for each component.